### PR TITLE
Issue/screens

### PR DESCRIPTION
**Pull request by kevvo15**
Routing between the currently necessary screens using stack navigation and drawer navigation.

### CODE REVIEW

File: components/Card.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+export default function Card(props) {
+    return (
+        <View style={styles.card}>
+            <View style={styles.cardContent}>
+                { props.children }
+            </View>
+        </View>
+    );
+};
```
Comment (CTOverton) [inline components/Card.js:12]: I noticed you did some props destructuring in some components and this one you didn't 

```suggestion
export default function Card({children}) {
    return (
        <View style={styles.card}>
            <View style={styles.cardContent}>
                { children }
            </View>
        </View>
    );
};
```

Aside from specifying types with typescript or using the PropTypes library, this is a good way to help your IDE figure out what properties your component has available. 

> Learn more: [Destructuring in JS](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)

File: screens/Albums.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import { globalStyles } from '../styles/global';
+import Card from '../components/Card';
+import { FlatList } from 'react-native-gesture-handler';
```
Comment (CTOverton) [inline screens/Albums.js:5]: This is why I like using the Webstorm IDE haha, so sometimes you'll find that the automated imports pull from whatever they think you want to use. That being said, while the FlatList is a part of the `react-native-gesture-handler` package, more likely you meant to grab the component from `react-native`

```suggestion
import { StyleSheet, Text, TouchableOpacity, View, FlatList } from 'react-native';
import { globalStyles } from '../styles/global';
import Card from '../components/Card';
```

File: screens/Home.js [hunk 3]
```diff
@@ -1,12 +1,36 @@
...
+        { section: 'ARTISTS', key: '3'}
+    ])
+
     return (
         <View style={globalStyles.container}>
             <Text style={globalStyles.text}>HOME SCREEN</Text>
+            <FlatList
+            data={sections}
+                renderItem={({ item }) => (
+                    <TouchableOpacity onPress={() => {
```
Comment (CTOverton) [inline screens/Home.js:20]: There is a new kid in town called [Pressable](https://reactnative.dev/docs/pressable) ;)
```suggestion
                    <Pressable onPress={() => {
```

File: screens/Albums.js (hunk 2 above)
Comment (kevvo15) [reply screens/Albums.js:5]: Thanks, I did notice this happened a couple of times and I fixed a few, must have missed this one. Looks like I'll have to look into getting Webstorm. Does Webstorm IDE help with identifying the proper import package?

**Pull request merged**